### ISSUE TITLE
Audio component

### ISSUE THREAD

**Issue by srallen** [grommet/grommet#2637]
### Expected Behavior
A component for audio similar to the Video component which renders the player, controls, and be styled.

### Actual Behavior
Grommet doesn't have a component for HTML 5 audio.

### Steps to Reproduce

1. 2. 3.

### Your Environment

- Grommet version:
- Browser Name and version:
- Operating System and version (desktop or mobile):

**Comment by oorestisime** [comment]
i would like to play a bit with that. is there a specific procedure to add a new component? (styling wise and feature wise?)

**Comment by krismorf** [comment]
@oorestisime What if it's exactly the same as the video component without the poster image?
![audio](https://user-images.githubusercontent.com/42035849/50835072-5ecf2c80-135e-11e9-8221-61ce01518c5f.jpg)

**Comment by oorestisime** [comment]
* i am wondering if the sound controls should be displayed directly (since there are no other options on the sound tag). 
* maybe most logic around controls should be abstracted and have a media component where  you can activate a video or an audio. i.e

```
<Media audio>
<source...>
</Media>

<Media video>
<source ...>
</Media>
```
The play/pause/stop louder, quieter mute etc controls should have the same logic.

**Comment by krismorf** [comment]
- In addition to the sound controls there is also a download button ([W3 Reference](https://www.w3schools.com/html/tryit.asp?filename=tryhtml5_audio_all)). But it's true that a volume control button on the main interface that also mutes the sound will provide better UX.
- I agree with this point for better DX. Could be something like: 
```jsx
<Player 
  source={}
  audio //or video
  ...
/>

**Comment by srallen** [comment]
Could we make the download button, and possibly the other controls, configurable? I realize that savvy users can figure out how to download the media anyway, but I'm tasked with making sure that it doesn't look like we're promoting the ability to download in the UI, which means disabling it or hiding the button if possible. Chromium added the non-spec `controlsList` attribute to support configuring this: https://bugs.chromium.org/p/chromium/issues/detail?id=675596, so something that works cross browser would be beneficial.

**Comment by oorestisime** [comment]
fwiw on FF i can't see any download button.

**Comment by srallen** [comment]
That's correct. FF never exposed the download button like Chrome did. I just mean a cross browser way to configure the controls, like `controlsList`, since the other browser engines haven't adopted it yet.

**Comment by ShimiSun** [comment]
We are working on putting together cool designs for this component, thanks for the great idea!

**Comment by ShimiSun** [comment]
@jensy would you like to share your design for the audio component with the community and get some feedback?

**Comment by jensy** [comment]
<img width="841" alt="image" src="https://user-images.githubusercontent.com/105723/51766965-3d2daf00-2099-11e9-9a80-28c8632535e5.png">

Here is a first sketch for the audio player component, it heavily leans on the HTML5 audio spec. Please let me know what you think!

**Comment by Lance-DC** [comment]
Are there any plans to move forward with this audio component?

**Comment by jcfilben** [comment]
Hi @Lance-DC currently we have no short term plans to add this component. If this is something you would be interested in using it would be helpful if you could share your use case so we can understand a little better how you are wanting to use this component as this could help us inform the direction we take.
